cluster/juju/layers/kubernetes-worker/layer.yaml

```diff
@@ -28,3 +28,4 @@ options:
     server_key_path: '/root/cdk/server.key'
     client_certificate_path: '/root/cdk/client.crt'
```

Comment:
Per our conversation: It appears that these client keys are not required for immediate availability of the application, however they are still required for some of the componentry.